tests/contrib/pytest/test_pytest.py

```diff
@@ -1159,10 +1159,16 @@ def test_ok(self):
         rec.assertoutcome(skipped=1, passed=1)
         spans = self.pop_spans()
         test_span_skipped = spans[0]
-        test_span_ok = spans[1]
-        test_suite_span = spans[4]
-        test_session_span = spans[2]
-        test_module_span = spans[3]
+        if len(spans) >= 6:
```

Comment:
I don't think we should have a scenario where we expect the number of spans to be able to change.